docker/README.md

```diff
@@ -13,7 +13,7 @@ Static binaries building using custom alpine image.
 [`docker-static-build.sh`](docker-static-build.sh) will build tezos binaries
 image defined in [Dockerfile](build/Dockerfile). In order to build them, just run the script
 ```
-./docker-static-build.sh
+cd .. && ./docker/docker-static-build.sh
```

Comment:
I slightly dislike that now Docker-way is not completely nix-less. `nix/sources.json` is a simple JSON file, but imo it still complicates the process a bit for foreign users. What if we pass `.tezos.ref` as a command line argument or env variable instead? And do `$(cat /sources.json | jq -r '.tezos.ref')` in the CI config?

Reply:
Well, docker packaging also uses `nix/sources.json` to determine sources version.
> What if we pass .tezos.ref as a command line argument or env variable instead?

Thus we'll again have two places when the version is defined. I think that we can extract tezos sources json to the root and modify `nix/sources.nix` to use this file, but then it won't be possible to use `niv` for updating

Reply:
> Thus we'll again have two places when the version is defined.

No, it will be in only one place: in `nix/sources.nix`. And CI config will be doing something like `$(cat nix/nix/sources.json | jq -r '.tezos.ref')`. So basically we will move the place where we get this version from `docker/docker-static-build.sh` to CI config. This way people who want to run this script on their machine will be able to do it without bothering about `nix/nix/sources.nix`.

Reply:
Ah, okay, I didn't get what you've been proposing

Reply:
Changed to env variable usage